app/templates/emails/brief_response_submitted.html

```diff
@@ -2,18 +2,72 @@
 <html>
 <head lang="en">
     <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
+
+    <style>
+    body {
+      font-family: Open Sans, sans-serif;
+    }
+
+    .notice {
+      padding: 0rem;
+      border: 2px solid #007554;
+      font-size: 2rem;
+    }
+
+    .notice p {
+      background: white;
+      margin: 0;
+    }
+
+    .notice span {
+      background: #007554;
+      padding: 1rem;
+      display: inline-block;
+      line-height: 2rem;
+      width: 3rem;
+      margin-right: 1rem;
+    }
+
+    .notice span span {
+      text-align: center;
+      width: 2rem;
+      background: white;
+      padding: 0.5rem;
+      display: inline-block;
+      color: #007554;
+      border-radius: 2rem;
+
+    }
+
+    .emphasised {
+      padding: 2rem;
+      background: #def4f9;
+    }
+
+    </style>
```

Comment:
Gmail will strip `style` tags, so should inline style this CSS

Handy Do's and Don'ts: https://www.campaignmonitor.com/css/